src/components/NewWorkspaceModal.js

```diff
@@ -63,7 +64,7 @@ const ariaInvalidBillingAccountMsg = (invalidBillingAccount) => {
 
 const NewWorkspaceModal = withDisplayName(
   'NewWorkspaceModal',
-  ({ cloneWorkspace, onSuccess, onDismiss, customMessage, requiredAuthDomain, title, buttonText, workflowImport }) => {
+  ({ cloneWorkspace, onSuccess, onDismiss, customMessage, requiredAuthDomain, requireEnhancedBucketLogging, title, buttonText, workflowImport }) => {
```

Comment:
`requireEnhancedBucketLogging` is supposed to be that the new workspace MUST have enhanced logging no matter what, right?  testing out this branch locally, if I select an authorization domain, then remove it, the checkbox becomes unchecked.  at least for my purposes, i'd want it to always be checked whether or not they have an authorization domain.  what's the intended behavior?

Reply:
My [upcoming PR](https://github.com/DataBiosphere/terra-ui/pull/4046) relies on this one; I got the behavior I want by adding a `requireEnhancedBucketLogging` [check](https://github.com/DataBiosphere/terra-ui/blob/a75a77735aebaaeb1102ff4fc9ddf773322671d0/src/components/NewWorkspaceModal.js#L394) to whether the checkbox is checked or not.  Is that correct behavior?  I have this branch merged into mine but don't know this one should merge into dev first.

Reply:
Am I understanding correctly that you're seeing this behavior on a workspace that starts with enhanced bucket logging enabled by default, then if you select an auth domain, then deselect an auth domain, the checkbox becomes unchecked?

Reply:
Yep!  I pass in `requireEnhancedBucketLogging = true` to the `NewWorkspaceModal` and then I am able to remove it by selecting and deselecting an auth domain.

Reply:
Hm that does seem like a bug to me, let me take a look!

Reply:
@calypsomatic please double check but I think I fixed this! It looks like the change from deselecting an auth domain was updating the checkbox despite it being disabled. I added an additional check for `requireEnhancedBucketLogging` so it should respect that now and have the expected behavior

Reply:
Looks great!  Thank you!